packages/imperative/src/OverridesLoader.ts

```diff
@@ -35,37 +35,7 @@ export class OverridesLoader {
       packageJson: any
   ): Promise<void> {
     // Initialize the Credential Manager
-    await (ImperativeConfig.instance.config?.exists ? this.loadCredentialManager : this.loadCredentialManagerOld)(config, packageJson);
-  }
-
-  /**
-   * Load the baked-in zowe CredentialManager and initialize it.
-   * If we need to reinstate 3rd party overrides, delete this function and
-   * rename loadCredentialManagerOld.
-   *
-   * @internal
-   * @param {IImperativeConfig} config - the current {@link Imperative#loadedConfig}
-   * @param {any} packageJson - the current package.json
-   */
-  public static async loadCredentialManager(
```

Comment:
👍 Thanks for combining the two functions `loadCredentialManager` and `loadCredentialManagerOld` 😋

Reply:
Something we'll probably need to discuss as a team: I'm not sure if combining them was totally a good idea. We may need to undo some of these changes to split out some logic again. 

Currently in the "next" branch, in the case that team config files do not exist, we maintain our old behavior of enabling secure credentials only when the CredentialManager setting is defined.

The implementation in this PR is that secure by default always rules. Which means if you have old school profiles stored in plain text, updating to Zowe v2 would break you. And there's no longer a `zowe scs update` command to easily unbreak you, since we recommend uninstalling the SCS when upgrading to Zowe v2.